packages/plugin-autocapture-browser/src/helpers.ts

```diff
@@ -89,7 +90,10 @@ export const isNonSensitiveElement = (element: Element) => {
   const isContentEditable =
     element instanceof HTMLElement ? element.getAttribute('contenteditable')?.toLowerCase() === 'true' : false;
 
-  return !SENSITIVE_TAGS.includes(tag) && !isContentEditable;
+  // Check if element or any parent has data-amp-mask attribute
+  const hasMaskAttribute = element.closest('[' + constants.DATA_AMP_MASK_ATTRIBUTE + ']') !== null;
```

Comment:
We shouldn't add it here since the concept of a sensitiveElement isn't the same as an element with masked text

Reply:
removed